scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -94,9 +100,32 @@ export class SpreadsheetStandardizer {
     ) {
       output.bonus_available = true;
     }
+    if (this.lowIncomeThresholds && isPlausibleLowIncomeRow(record)) {
+      const low_income_program = this.retrieveLowIncomeProgram(
+        authorityName,
+        state,
+      );
+      if (low_income_program) {
+        output.low_income = authorityName;
+      } else if (low_income_program === undefined) {
+        // This is checked up front by the caller rather than
+        // at the row level to avoid spamming error messages.
+      } else {
+        console.log(
+          `Warning: no low-income thresholds found for ${record.id} despite references to income eligiblity in description or income restrictions columns. This either means: 1) it should use the 'default' thresholds, or 2) you need to define thresholds for it, or 3) it's not a low-income row and no action is necessary.`,
```

Comment:
What is the action for 1)? How would one use the 'default' thresholds, compared to defining a default threshold for it?

Want to make sure the warning message is as actionable and straightforward as possible.

Reply:
So the answer (and I'm fleshing out the comment as such) is that you just have to fill in 'default' manually for those rows. Which is too bad, since you'll have to do it every time you rerun the script. Not the best experience.

I don't see any better options, at least in the short-term. There really are three scenarios and the script can't know which one to use. I think we can keep this for now and keep it in mind as an area for future improvement – maybe we can find some way to specify in the spreadsheet that a row should use the default thresholds. Or maybe we do away with default thresholds and change the data structure so it's easier to share low-income thresholds across unrelated programs.